src/commands/ReminderCommand/ReminderCommand.js

```diff
@@ -0,0 +1,83 @@
+const BaseCommand = require('../../utils/structures/BaseCommand');
+const momentTimezone = require('moment-timezone');
+const { MessageCollector } = require('discord.js');
+
+const scheduledSchema = require('../../models/scheduled-schema');
+
+module.exports = class ReminderCommand extends BaseCommand {
+  constructor() {
+    super('schedule', 'textAnswers', []);   // change the name to remind later
```

Comment:
a `schedule help` command would be very useful, which could also be run when only writing `schedule`

Reply:
Since I have another branch where I add new features to this command, I will add this in that following branch